Add required accountId parameter to JMAP methods

cketti opened the pull request:
I'm not sure about the API. But we have to do something similar to this in order to supply the mandatory `accountId` property to JMAP method calls.

@iNPUTmice: What do you think?

iNPUTmice commented:
Yes I think we can do it like this.
I had been hoping to come up with a solution that has less boiler plate injecting the account id everywhere; but seems like this is not very feasible.

Very minor nitpick: I don’t think we want to expose methods like `getPrimaryMailAccountId()` for every possible capability. Just proxiying the newly created getPrimaryAccount(Class) is probably good enough. (Since the library is extensible we won’t be able to provide those more direct methods for all caps anyway.)

cketti commented:
I removed `getPrimaryMailAccountId()` and switched to `getPrimaryAccount(Class)` like you suggested.

I went through the rest of the methods and added the `accountId` parameter to the constructors.
